cypress/pages/ndv.ts

```diff
@@ -199,10 +199,11 @@ export class NDV extends BasePage {
 			getVisiblePopper().find('li').last().click();
 		},
 
-		setInvalidExpression: (fieldName: string, invalidExpression?: string) => {
+		setInvalidExpression: ({ fieldName, invalidExpression, delay }: { fieldName: string, invalidExpression?: string, delay?: number }) => {
```

Comment:
Nitpicking here but IMO this method should have same signature as `typeIntoParameterInput` since it has very similiar inputs.  
```ts
setInvalidExpression: (
			fieldName: string,
			invalidExpression: string,
			opts?: { delay?: number },
		) => {
			this.actions.typeIntoParameterInput(fieldName, '=');
			this.actions.typeIntoParameterInput(fieldName, invalidExpression ?? "{{ $('unknown')", {
				parseSpecialCharSequences: false,
				delay: opts?.delay,
			});
			this.actions.validateExpressionPreview(fieldName, `node doesn't exist`);
		},
		```

Reply:
Agree but I also think it makes sense for them to have different signatures since they are used for different purposes. For example, `setInvalidExpression` doesn't care about `parseSpecialCharSequence`